src/Command/RandomIssueCommand.php

```diff
@@ -86,13 +86,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $randomIssue = $searchIssueModel->getItems()[$randomIssueIndex];
 
+        $io->section($randomIssue->getTitle());
+
+        $io->newLine(1);
+
```

Comment:
Is there a reason to keep this? The title is already in the table so I don't think it needs to appear twice.

Alternatively, if it looks better, we could remove the title from the table and have the title as a heading instead.

Reply:
You are correct, I didn't see that. The additional section here is not needed.